docs/configs/bebop.yaml

```diff
@@ -16,14 +18,14 @@ engine:
         # string to prepend to #SBATCH blocks in the submit
         # script to the scheduler
         # eg: '#SBATCH --constraint=knl,quad,cache'
-        scheduler_options: {{ OPTIONS }}
+        # scheduler_options: {{ OPTIONS }}
```

Comment:
Uncomment this line?

Reply:
It was previously uncommented and I wasn't sure what to set. So I commented it out when testing and it wasn't necessary as my tasks ran. As it is optional I considered deleting it entirely, but thought it may be useful to leave? What do you think?

Reply:
Hmm, I suspect this line was copied from a nersc machine, I would remove it entirely including the comment block above it.